.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Problem
```

Comment:
Perhaps "Issue" rather than "Problem", to tie to the agile terminology and Jira integration.